### ISSUE TITLE
QRCode Information causes overflow on mobile screens

### ISSUE THREAD

**Issue by fairhat** [prechelt/anwesende#2]
The example page for QRCodes causes the body element to overflow on mobile screens.

Suggestions:
Assuming the ui library is bootstrap the qrcode area could be wrapped in responsive grid classes

Docs:
https://getbootstrap.com/docs/5.0/layout/grid/#stacked-to-horizontal

I could also create a PR to fix this.
Are there any contribution guidelines?

Attached video demonstrating the issue:

https://box.fu-berlin.de/s/ABtJQbMdyjWRFa4

**Comment by prechelt** [comment]
Thanks for the information!
I had indeed not considered this.
However, I feel there is no better solution:
Changing the layout (putting the QR code above the text block on its right) is out of the question: The idea of the page is to show what the QR codes look like as they will be pasted at each seat.
So the only option would be to make the thing smaller, but then it would become illegible. (Also, the user can do this themselves at any time.)
I suggest we leave that as it is.
There is also a technical problem, because the current layout is realized as a table, because all other solutions I have tried created problems with the page-breaking during the PDF conversion.
